src/nvim/types.h

```diff
@@ -40,8 +40,9 @@ typedef struct {
   int OffsetRow;
   int OffsetColumn;
 
-  int internal_Rows;
-  int internal_Columns;
+  // the size expected to be allocated to the internal grid
+  int InternalRows;
```

Comment:
newly introduced variables should use `lower_case` (previously global varables moved to a struct is more of a border case, but that is something that we can deal with later)